ops/model.py

```diff
@@ -606,6 +606,9 @@ def open_port(self, protocol: typing.Literal['tcp', 'udp', 'icmp'],
         However, normally charms should make the same open_port() call from
         every unit.
 
+        Use :func:`set_ports()` for a more declarative approach where all of
```

Comment:
It doesn't seem to matter to Sphinx, but elsewhere we've used `:meth:` instead of `:func:` and dropped the parens on the name, so let's do that for consistency:
```suggestion
        Use :meth:`set_ports` for a more declarative approach where all of
```